### PR TITLE
[ci] Fix Linux x64 Bitcode job

### PR DESCRIPTION
**Pull request by luhenry**

### CODE REVIEW

**Comment by luhenry** [comment]
@monojenkins build Linux x64 Bitcode

**Comment by luhenry** [comment]
@monojenkins build Linux x64 Bitcode

**Comment by luhenry** [comment]
@monojenkins build Linux x64 Bitcode

**Comment by luhenry** [comment]
@monojenkins build Linux x64 Bitcode

**Comment by luhenry** [comment]
@monojenkins build Linux x64 Bitcode

**Comment by akoeplinger** [comment]
@monojenkins build Linux x64 Bitcode

**Comment by luhenry** [comment]
@monojenkins build Linux x64 Bitcode

**Comment by luhenry** [comment]
@monojenkins build Linux x64 Bitcode

**Comment by luhenry** [comment]
@monojenkins build Linux x64 Bitcode
@monojenkins build Linux ARMv7 Bitcode

**Comment by luhenry** [comment]
@monojenkins build Linux ARMv7 Bitcode

**Comment by luhenry** [comment]
@monojenkins build Linux x64 Bitcode

**Comment by luhenry** [comment]
@monojenkins build Linux ARMv7 Bitcode

**Comment by luhenry** [comment]
@monojenkins build failed

**Comment by luhenry** [comment]
@monojenkins build Linux x64 Bitcode
@monojenkins build Linux ARMv7 Bitcode

**Comment by luhenry** [comment]
@monojenkins build Linux x64 Bitcode
@monojenkins build Linux ARMv7 Bitcode

**Comment by luhenry** [comment]
@monojenkins build Linux x64 Bitcode

**Comment by luhenry** [comment]
@monojenkins build Linux ARMv7 Bitcode

**Comment by luhenry** [comment]
@monojenkins build Linux x64 Bitcode

**Comment by luhenry** [comment]
@monojenkins build Linux ARMv7 Bitcode

**Comment by luhenry** [comment]
@monojenkins build Linux x64 Bitcode

**Comment by luhenry** [comment]
@monojenkins build Linux x64 Bitcode

**Comment by luhenry** [comment]
@monojenkins Linux x64 Interpreter+Bitcode

**Comment by luhenry** [comment]
@monojenkins build Linux x64 Interpreter+Bitcode

**Comment by luhenry** [comment]
@monojenkins build Linux x64 Bitcode

**Comment by luhenry** [comment]
@monojenkins build Linux x64 Bitcode+Interpreter

File: configure.ac [hunk 1]
```diff
@@ -1228,9 +1228,23 @@ elif test x$with_runtime_preset = xbitcode; then
...
+   enable_cooperative_suspend_default=yes
+   TEST_PROFILE=testing_aot_bitcode_interp
+   enable_llvm_default=yes
+
+   # mscorlib.dll aot compilation crashes
+   mono_feature_disable_com='yes'
+
+   # We need at least clang++-6.0 to read the LLVM IR produced by the runtime
+   AOT_BUILD_FLAGS="--aot=llvmonly,interp,clangxx=clang++-6.0,$INVARIANT_AOT_OPTIONS"
+   AOT_RUN_FLAGS="--llvmonly-interp"
```
Comment (lewurm) [inline configure.ac]: should we also have `--runtime=mobile` here for `AOT_BUILD_FLAGS` and `AOT_RUN_FLAGS`? similar to the `bitcode` profile

Comment (luhenry) [reply configure.ac]: It's not set for `fullaotinterp` which is why I didn't do it here.

**Comment by luhenry** [comment]
@monojenkins build Linux x64 Bitcode

**Comment by luhenry** [comment]
@monojenkins build Linux x64 Bitcode+Interpreter

**Comment by akoeplinger** [comment]
@monojenkins backport 2019-02

**Pull request merged**